src/org/zaproxy/zap/extension/alert/ExtensionAlert.java

```diff
@@ -603,9 +603,12 @@ void recalcAlerts() {
                 int alertId = v.get(i).intValue();
                 RecordAlert recAlert = tableAlert.read(alertId);
                 Alert alert = new Alert(recAlert);
-                if (!allAlerts.contains(alert)) {
-                    allAlerts.add(alert);
-                }
+				if (alert.getHistoryRef() != null) {

```

Comment:
@thc202 I'm not sure why that happened, i'm looking at my file right now and the `if` is in line and properly indented :S